src/avitab/apps/MapApp.cpp

```diff
@@ -177,7 +157,11 @@ void MapApp::setMapSource(MapSource style) {
         selectNavigraph(maps::NavigraphMapType::WORLD);
         break;
     case MapSource::ONLINE_TILES:
-        selectOnlineMaps();
+        if (init) {
+            selectOnlineMaps(false);

```

Comment:
The first time we call setMapSource with init == true, we call the `selectOnlineMaps` in non-interactive mode.